.browserslistrc

```diff
@@ -1,3 +1,5 @@
 > 1%
 last 2 versions
 not dead
+not IE 11
+Firefox ESR
```

Comment:
I'm all for IE eradication, but I don't know if I want to specify this LTS version of FF specifically. If this version is used by more than 1% of the user base it will be added in the list, no?

IMHO, the least we customize, the better